docs/class5/connexml.md

```diff
@@ -417,6 +428,62 @@ An existing call is transferred to a different destination using the `Dial` ver
             </Dial>
         </Response>
         ```
+    
+    10. **Voicemail**
+        ``` xml
+        <?xml version="1.0" encoding="UTF-8"?>
+        <Response>
+            <Dial>
+                <Voicemail />
+            </Dial>
+        </Response>
+        ```
+    
+    11. **Voicemail Inbox**
+        ``` xml
+        <?xml version="1.0" encoding="UTF-8"?>
+        <Response>
+            <Dial>
+                <Voicemail inbox="true">
+                 <Voicemail/>
+            </Dial>
+        </Response>
+        ```
+
+    12. **Barge (Default Method)**
+        ``` xml
+        <?xml version="1.0" encoding="UTF-8"?>
+        <Response>
+            <Dial>
+                <Barge></Barge>
+            </Dial>
+        </Response>
+        ```
+    
+    13.  **Barge**
+        ``` xml
+        <?xml version="1.0" encoding="UTF-8"?>
+        <Response>
+            <Dial>
+                <Barge>
+                    whisper="a" // enables whisper mode in aleg
+                    bridge="b" // enables listen to in bleg
+                    commands= "false" // false means no DTMF signals/commands during eavesdrop
+                </Barge>
+            </Dial>
+        </Response>
```

Comment:
<?xml version="1.0" encoding="UTF-8"?>
<Response>
    <Dial>
        <Barge
            whisper="a"                <!-- Enables whisper mode in the A leg-->
            bridge="b"                 <!-- Enables listen to B leg -->
            commands="false"    <!-- Disables DTMF signals/commands during barge -->
        ></Barge>
    </Dial>
</Response>

Reply:
done